### PR TITLE
Oceproject 4852

### PR DESCRIPTION
**Pull request by SupriyaSaxena0605**
Please approve the test for OCEPROJECT-4852.
This ticket includes test method 'verifySitewideSearchBottomSearchBar'.

### CODE REVIEW

File: src/main/java/gov/nci/commonobjects/SitewideSearch.java [hunk 1]
```diff
@@ -77,16 +82,26 @@ public WebElement getBestBetBox() {
...
 	// Get Search Results page Search Within Search Box
 	public WebElement getSearchWithinSearchBox() {
+		ScrollUtil.scrollIntoview(driver, driver.findElement(By.xpath("(//h4)[2]")));
 		return txt_SearchWithinSearchBox;
 	}
 
 	public WebElement getSearchWithinSearchButton() {
-		return btn_SearchWithinSearch;
+
+		ScrollUtil.scrollIntoview(driver, driver.findElement(By.xpath("(//h4)[2]")));
```
Comment (blairlearn) [inline src/main/java/gov/nci/commonobjects/SitewideSearch.java:100]: Please replace xpath with css.

File: src/test/java/gov/nci/CommonObjects/Tests/SitewideSearch_Test.java [hunk 2]
```diff
@@ -40,20 +42,53 @@ public void setup(String browser) throws MalformedURLException {
...
 		logger.log(LogStatus.PASS, "Verify that Search button are displayed");
 	}
 
-	@Test(dataProvider = "Search", groups = { "Smoke" })
+	// Testing the Bottom search bar on Search results page
+	@Test(groups = { "Smoke" }, dataProvider = "Search", priority = 3)
+	public void verifySitewideSearchBottomSearchBar(String keyword) {
+
+		System.out.println("Search Keyword: " + keyword);
+		search.search(keyword);
```
Comment (blairlearn) [inline src/test/java/gov/nci/CommonObjects/Tests/SitewideSearch_Test.java:63]: Methods on a page object which perform an action should return a new page object.  Assertions are then executed against the returned object.

This test is actually concealing multiple other tests.

* If a search is performed, does the browser move to the expected new URL?
  * The test should check for the expected URL path.
  * The test should also check for the expected URL parameters.
* Load a search results page (use the explicit URL) and check (each of these is a separate test)
  * Does the search results page contain a New Search button?
  * Does the search results page contain a Search within results button?
  * etc.

The tests for the search results page can be parameterized to load multiple URLs from a data file.

**Review by blairlearn — CHANGES_REQUESTED** [review]
See notes in the conversation view.

**Comment by SupriyaSaxena0605** [comment]
Created separate pull requests for individual tickets, therefore closing this one

**Pull request closed without merging**